src/Serialization/main.jl

```diff
@@ -61,7 +61,7 @@ const typeMap = Dict{Type, String}([
     Nemo.GaloisFmpzField => "Nemo.GaloisFmpzField",
     fmpz => "fmpz",
     gfp_elem => "Nemo.gfp_elem",
-    gfp_fmpz_elem => "Nemo.gfp_fmpz_elem",
+    gfp_fmpz_elem => "Nemo.gfp_fmpz_elem"
```

Comment:
Please leave the trailing comma, it results in smaller differs when we later add more entries to this dict.

Also, did you mean to hook up the `fmpq_mpoly` code here?

Reply:
I wasn't sure what exactly this map was for, and seemed to work without so i removed the line I wrote and forgot that comma was already there on line above. After reading the comment about decodeType, I see that this map provides a speed so I have added fmpq_mpoly